test/axes/numericAxisTests.ts

```diff
@@ -431,4 +431,36 @@ describe("NumericAxis", () => {
     svg.remove();
   });
 
+  it("reasonably approximates tick label sizes with approximate measuring", () => {
+    var SVG_WIDTH = 500;
+    var SVG_HEIGHT = 100;
+
+    [[-1, 1],
+     [0, 10],
+     [0, 999999999]
+    ].forEach((domainBounds) => {
+      var scale = new Plottable.Scales.Linear();
+      scale.domain(domainBounds);
+
+      var svgApprox = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var numericAxisApprox = new Plottable.Axes.Numeric(scale, "left");
+      numericAxisApprox.usesTextWidthApproximation(true);
+      numericAxisApprox.renderTo(svgApprox);
+      var widthApprox = (<any> numericAxisApprox)._computeWidth();
+
+      var svgExact = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var numericAxisExact = new Plottable.Axes.Numeric(scale, "left");
+      numericAxisExact.usesTextWidthApproximation(false);
+      numericAxisExact.renderTo(svgExact);
+      var widthExact = (<any> numericAxisExact)._computeWidth();
+
+      svgApprox.remove();
+      svgExact.remove();
+
+      assert.isTrue(widthApprox < (widthExact * 1.55),
+        "an approximate scale of [" + domainBounds[0] + "," + domainBounds[1] + "] is less than 55% larger than an exact scale");
+      assert.isTrue(widthApprox >= widthExact,
+        "an approximate scale of [" + domainBounds[0] + "," + domainBounds[1] + "] is smaller than an exact scale");
```

Comment:
There is an `assert.operator` for usage with `<` and `>=`.  Please use that instead of `assert.isTrue`